src/components/UserPlan.tsx

```diff
@@ -36,7 +36,7 @@ export const UserPlan = ({ user }: UserPlanProps) => {
   } = sms;
 
   const isPlanTypeMonthlyDeliveries = planType === "monthly-deliveries";
-  const isPlanTypeSuscribers = planType === "suscribers";
+  const isPlanTypeSubscribers = planType === "subscribers";
```

Comment:
Por alli estaría bueno tener tambien una constante, que opinás? de tal forma que `"subscribers"` lo puedas referenciar usando algo como esto `PLAN_TYPE.byContact`

Reply:
Creo que no es necesario porque `planType` ahora está tipado.

Reply:
Seria algo así como, `PLAN_TYPE.subscribers`?

Reply:
Yo decia algo como: 
`const isPlanTypeSubscribers = planType === PLAN_TYPE. byContact;`

Reply:
@jcamposmk yo creo que no es necesario definir una constante `PLAN_TYPE. byContact` o un _enum_ porque estamos usando [union de tipos literales](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types) y eso ya nos da _intellisense_ y _type safety_:

![image](https://user-images.githubusercontent.com/1157864/190180139-3238e0d2-2f3a-4fa4-a900-5b835d1985b6.png)

![image](https://user-images.githubusercontent.com/1157864/190180168-5e28fbe8-78de-42b5-a344-bf0a2c5d481d.png)

Por otro lado, [ya no se recomienda utilizar _enums_](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#enums):

> _Unlike most TypeScript features, this is not a type-level addition to JavaScript but something added to the language and runtime. Because of this, it’s a feature which you should know exists, but maybe hold off on using unless you are sure_